src/app/services/news.service.ts

```diff
@@ -0,0 +1,57 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable, from } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { News } from '../models/News';
+
+export interface NewsService {
+  getNews(): Observable<News[]>;
+}
+
+@Injectable({
+  providedIn: 'root'
+})
+export class DummyNewsService implements NewsService {
+  getNews(): Observable<News[]>  {
+    return from([[
+      {
+        url: 'http://www.google.com',
+        title: 'COVID kills',
+        description: 'Dumb ppl still not wearing masks.'
+      },
+      {
+        url: 'http://www.bridgebase.com/v3',
+        title: 'Moron dies after injecting himself with bleach',
+        description: 'Trump claims injecting bleach as a COVID treatment is "very, very bad"'
+      }
+    ]]);
+  }
+}
```

Comment:
This might be unnecessary if its just for unit tests? You're able to mock out the response from the HttpClient to return a known response, see [Testing HTTP Services](https://angular.io/guide/testing-services#testing-http-services)

Reply:
Originally I was just experimenting with dependency injection, so wrote 2 classes to make sure I can swap easily between them.
As you pointed out, it's not necessary.